"make login"時のエラー対応

下記エラーの対応
```
$ make login
Configure aws sso ...
Configured aws sso

Error loading SSO Token: Token for https://d-9567183e63.awsapps.com/start does not exist
make: *** [makefile:28: finally] Error 255
```

--- a/makefile
+++ b/makefile
@@ -1,3 +1,6 @@
+include project.env
+SHELL=/bin/bash
+
 login: configure sso-login mfa finally
 	@aws-export-credentials --credentials-file-profile amplify
 

